README.md

```diff
@@ -421,6 +421,7 @@ that caused Neoformat to be invoked.
 - TypeScript
   - [`tsfmt`](https://github.com/vvakame/typescript-formatter),
     [`prettier`](https://github.com/prettier/prettier),
+    [`prettier-eslint`](https://github.com/kentcdodds/prettier-eslint-cli),
```

Comment:
could you also update the `docs/neoformat.txt` so the docs in vim show this as well?